rotkehlchen/tests/external_apis/test_etherscan.py

```diff
@@ -90,3 +96,45 @@ def test_deserialize_transaction_from_etherscan():
         input_data=bytes.fromhex(data['input'][2:]),
         nonce=0,
     )
+
+
+def test_genesis_deserialize_transaction():
+    """Test that the genesis block transactions are deserialized properly"""
+    data = {'blockNumber': 0, 'timeStamp': 1439048640, 'hash': 'GENESIS_9c81f44c29ff0226f835cd0a8a2f2a7eca6db52a711f8211b566fd15d3e0e8d4', 'nonce': 0, 'from': 'GENESIS', 'to': '', 'value': 100, 'gas': 0, 'gasPrice': 0, 'isError': 0, 'txreceipt_status': '', 'input': '0x313233', 'contractAddress': '0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae', 'cumulativeGasUsed': 0, 'gasUsed': 0, 'confirmations': 8569454}  # noqa: E501
+    transaction = deserialize_ethereum_transaction(data, internal=False, ethereum=None, trace_id=1)
+    tx_hash = deserialize_evm_tx_hash('0x0000000000000000000000000000000000000000')
+    from_address = deserialize_ethereum_address('0x0000000000000000000000000000000000000000')
+
+    assert transaction == EthereumInternalTransaction(
+        parent_tx_hash=tx_hash,
+        trace_id=1,
+        timestamp=1439048640,
+        block_number=0,
+        from_address=from_address,
+        to_address=None,
+        value=100,
+    )
+
+
+def test_etherscan_get_transactions_genesis_block(etherscan: Etherscan):
+    """Test that the genesis transactions are correctly returned"""
+    assert etherscan.db is not None
+
+    etherscan.db.add_blockchain_accounts(
+        blockchain=SupportedBlockchain.ETHEREUM,
+        account_data=[
+            BlockchainAccountData('0x756F45E3FA69347A9A973A725E3C98bC4db0b5a0'),
+        ],
+    )
+    transactions = etherscan.get_transactions(
+        deserialize_ethereum_address('0x756F45E3FA69347A9A973A725E3C98bC4db0b5a0'),
+        action='txlist',
```

Comment:
You are working with internal transactions so I would say that `txlistinternal` is better. Isn't it?

Reply:
Hm, I'm testing if it'll return 0x0 hash, so it's fine, cause that is intended, maybe I'll create add a test to return internal transaction which should to test if internal txns were added.